testsuite/tests/lib-list/test.ml

```diff
@@ -0,0 +1,47 @@
+(* Standard test case *)
+let () =
+  let l = List.init 10 (fun x -> x) in
+  assert (List.exists (fun a -> a < 10) l);
+  assert (List.exists (fun a -> a > 0) l);
+  assert (List.exists (fun a -> a = 0) l);
+  assert (List.exists (fun a -> a = 1) l);
+  assert (List.exists (fun a -> a = 2) l);
+  assert (List.exists (fun a -> a = 3) l);
+  assert (List.exists (fun a -> a = 4) l);
+  assert (List.exists (fun a -> a = 5) l);
+  assert (List.exists (fun a -> a = 6) l);
+  assert (List.exists (fun a -> a = 7) l);
+  assert (List.exists (fun a -> a = 8) l);
+  assert (List.exists (fun a -> a = 9) l);
+  assert (not (List.exists (fun a -> a < 0) l));
+  assert (not (List.exists (fun a -> a > 9) l));
+  assert (List.exists (fun _ -> true) l);
+;;
+
```

Comment:
Um. Shouldn't the following suffice? We aren't testing `List.exists` here.
```ocaml
assert ((List.init 10 (fun x -> x)) = [ 0; 1; 2; 3; 4; 5; 6; 7; 8; 9 ])
```

Reply:
FWIW, yes we are!